docs/team/kimideas8.md

```diff
@@ -18,14 +18,14 @@ This application was created with the aim of helping NUS Computing and Engineeri
     - What it does: allows the user to add notes one at a time to a location of choice, view an existing list of notes by locations, and delete notes at a location of choice one at a time 
     - Justification: It provides users with way of making quick notes within the campus, and saves users the trouble of adding notes in their own note-taking softwares, especially if the user is rushing for time.
       Since it helps to naturally organise a person's notes by locations, the user can easily recall what the note was made for.
-      
+
+<div style="page-break-after: always;"></div>
```

Comment:
what does this do?

Reply:
ohhhh its a page break for the conversion to pdf file in github.io site

Reply:
Oh then don't we need this for the UG and DG as well?